pandas/core/frame.py

```diff
@@ -2149,7 +2151,12 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
                    compression=compression, index=index,
                    partition_cols=partition_cols, **kwargs)
 
-    @Substitution(header='Whether to print column labels, default True')
+    @Substitution(header='Whether to print column labels, default True',
+                  col_space_type='str or int',
+                  col_space='The minimum width of each column in CSS length '
+                            'units.  An int is assumed to be px units.\n\n'
+                            '            .. versionadded:: 0.25.0\n'
+                            '            Abillity to use str')
```

Comment:
I think you need a blank line here. does this render properly when this doc-page is built (meaning w/o warning)?

Reply:
Didn't need an extra blank line, but I did need to indent the line under `versionadded` a little more to remove warnings.  Should be all set now.  Here is what it looks like rendered:


![image](https://user-images.githubusercontent.com/25773476/55691636-2a273d00-596e-11e9-9ec0-4576cec43805.png)